<a href="https://colab.research.google.com/github/benoynoel/Fortnite2.github.io/blob/master/Hypothesis_Testing_Assignment(Chi_SquareTtest).ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

#Libraries

In [ ]:
import pandas as pd
import numpy as np
import scipy
from scipy import stats

#Data

In [ ]:
df = pd.read_csv("BuyerRatio.csv")
df

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

In [ ]:
data = np.array(df)
data

array([['Males', 50, 142, 131, 70],
       ['Females', 435, 1523, 1356, 750]], dtype=object)

In [ ]:
final_data = data[:,[1,2,3,4]]
final_data

array([[50, 142, 131, 70],
       [435, 1523, 1356, 750]], dtype=object)

#X^2-critical value

In [ ]:
scipy.stats.chi2.ppf(1-0.05,3)

7.814727903251179

#x^2-Statistical & p-value

In [ ]:
scipy.stats.chi2_contingency(final_data)

Chi2ContingencyResult(statistic=1.595945538661058, pvalue=0.6603094907091882, dof=3, expected_freq=array([[  42.76531299,  146.81287862,  131.11756787,   72.30424052],
       [ 442.23468701, 1518.18712138, 1355.88243213,  747.69575948]]))

since P-Value > Alpha value i.e, 0.6603 > 0.05. Hence all proportions are equal
